packages/web-components/@sfx/sayt/test/sayt.test.ts

```diff
@@ -22,6 +22,7 @@ describe('Sayt Component', () => {
       expect(addEventListener).to.be.calledWith('click', sayt.processClick);
       expect(addEventListener).to.be.calledWith('keydown', sayt.processKeyEvent);
       expect(addEventListener).to.be.calledWith(AUTOCOMPLETE_RECEIVED_RESULTS_EVENT, sayt.showCorrectSayt);
+      expect(addEventListener).to.be.calledWith('sfx::searchbox_change', this.requestSayt);
```

Comment:
In case we miss this later when you are adding more tests: we should also be asserting that `this.requestSayt` is removed in the `disconnectedCallback()` by adding to the test below.